search_index.rb

```diff
@@ -90,4 +90,4 @@
 puts "Indexing #{referendum_data.length} Referendums..."
 
 puts "total #{referendum_data.concat(contributor_data.concat(candidate_data)).length}"
```

Comment:
it's weird that the `puts` statement modifies the arrays, and that all data is included in the `referendum_data` array. I'd recommend for clarity:

```ruby
all_data = referendum_data + contributor_data + candidate_data
puts "total #{all_data.length} items"
index.replace_all_objects(all_data)
```